bedrock/firefox/templates/firefox/browsers/compare/safari.html

```diff
@@ -243,7 +243,7 @@ <h2>{{ ftl('compare-shared-portability-strong') }}</h2>
       <p>{{ ftl('compare-safari-firefox-and-safari-both-provide') }}</p>
 
       {% if switch('moz-accounts-update') %}
-        <p>{{ ftl('compare-safari-firefox-also-offers-a-similar-updated-v2', fallback='compare-safari-firefox-also-offers-a-similar-updated', attrs = 'href="%s" data-cta-type="link" data-cta-name="Firefox Account"'|safe|format(url('firefox.accounts'))) }}</p>
+        <p>{{ ftl('compare-safari-firefox-also-offers-a-similar-updated-v3', fallback='compare-safari-firefox-also-offers-a-similar-updated', attrs = 'href="%s" data-cta-type="link" data-cta-name="Firefox Account"'|safe|format(url('firefox.accounts'))) }}</p>
```

Comment:
Also, should `data-cta-name` now be `"Mozilla account"`?

Reply:
Ah, good looking out :)